ingest/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.pythonPath": "/usr/bin/python",
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true
+}
```

Comment:
Is this a tool parameter file (vscode)? Should it be ignored?

Reply:
yeah, this is vscode stuff, i decided to check that in because @ulgu3559 did with the docker image stuff and it helps. Otherwise it's ignored